aspnetcore/host-and-deploy/iis/advanced.md

```diff
@@ -48,6 +48,29 @@ The `disallowRotationOnConfigChange` setting is intended for blue/green scenario
 
 This setting corresponds to the API <xref:Microsoft.Web.Administration.ApplicationPoolRecycling.DisallowRotationOnConfigChange?displayProperty=nameWithType>
 
+## Fix for 503's during app recycle in IIS
```

Comment:
Not a fan of the title, here's some other thoughts:

Reduce 503 likelihood during app recycle
Delay managed app shutdown during app recycle
Delay server shutdown on recycle